src/usage_metrics/scripts/save_zenodo_metrics.py

```diff
@@ -33,6 +33,16 @@ class ZenodoStats(BaseModel):
     version_views: int
 
 
+class ZenodoMetadata(BaseModel):
+    """Pydantic model representing relevant Zenodo metadata.
+
+    See https://developers.zenodo.org/#representation.
+    """
+
+    version: str | None = None
+    publication_date: datetime = None
```

Comment:
nit: looks like `publication_date` is required, but `version` is not - we could protect ourselves from malformed metadata by not giving this the default `None` value. 

Also if you keep the ` = None`, the type should be `datetime | None` 🤷